libs/printing/package.json

```diff
@@ -8,7 +8,7 @@
   "main": "build/index.js",
   "types": "build/index.d.js",
   "scripts": {
-    "build": "tsc --build tsconfig.build.json",
+    "build": "playwright install chromium && tsc --build tsconfig.build.json",
```

Comment:
Seems like this should happen in the install step, no?

Reply:
I was just placeholding while I figure out the general strategy here. Need to talk to @amcmanus. 

I think that the best way might be to install chromium with `sudo apt` and specify a custom executable path in our code based on environment.

If we did the executable with the node package install, I think we'd need to set up some sort of `pnpm` post-install hook, as `install` is not defined by us it's just a `pnpm` command? But whether that works well or not, I think it would be bad for trusted build.

Reply:
debb4800555520d4130801e04b171257d8df5519

@jonahkagan I set it up so that if the system chromium executable exists before install, it uses that and does not try to install via Playwright. If it doesn't exist, it tries to install via Playwright.

This is a nice compromise because on online environments (`design/backend`, CircleCI) it will work regardless of whether we ran `sudo apt install chromium`. In an offline environment, it will try to use the pre-installed package and fail loudly trying to install via Playwright if it doesn't exist.

There could be concern about difference between the chromium binaries installed by `sudo apt` and by Playwright. But they appear to generate identical PDFs.

Reply:
Great!